tools/DOI/authors.py

```diff
@@ -239,7 +240,22 @@
     'William F Godoy'         : 'Godoy, William F',
     'Islam, Fahima F'         : 'Islam, Fahima',
     'Mathieu Tillet'          : 'Tillet, Mathieu',
-    'StephenSmith'            : 'Smith, Stephen'
+    'MathieuTillet'           : 'Tillet, Mathieu',
+    'StephenSmith'            : 'Smith, Stephen',
```

Comment:
is already above https://github.com/mantidproject/mantid/pull/29644/files#diff-29d12c5945d69452fc4830199bec6ff4L217

Reply:
This is just a side effect of the fact I had to add `MathieuTillet` with the other alias they used 
I haven't touched Stephen's mapping in this PR